pkg/app/component.go

```diff
@@ -64,6 +64,15 @@ type PreRenderer interface {
 	OnPreRender(Context)
 }
 
+// Initialiser is the interface that describes a component that can perform
+// additional initialisations once before it is the first time rendered.
```

Comment:
Initialiser is the interface that describes a component that can perform additional initialization before a component is mounted (or pre-rendered?)